base/reducedim.jl

```diff
@@ -688,7 +688,7 @@ minimum(A::AbstractArray; dims)
     minimum(f, A::AbstractArray; dims)
 
 Compute the minimum value from of calling the function `f` on each element of an array over the given
```

Comment:
`Compute the minimum value from of calling`
Not introduced here, but: "from of calling"